NOTICE

```diff
@@ -1,5 +1,64 @@
 Apache Hudi (incubating)
-Copyright 2020 The Apache Software Foundation
+Copyright 2019-2020 The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies
+
+Metrics
+Copyright 2010-2013 Coda Hale and Yammer, Inc.
+
+This product includes software developed by Coda Hale and Yammer, Inc.
+
+-----------------
+Guava
+Copyright (C) 2007 The Guava Authors
+
+Licensed under the Apache License, Version 2.0
+
+-----------------
+Kryo (https://github.com/demurgos/kryo)
```

Comment:
https://github.com/EsotericSoftware/kryo/blob/master/LICENSE.md is the actual thing?